pythonforandroid/bootstraps/sdl2/build/templates/AndroidManifest.tmpl.xml

```diff
@@ -132,6 +132,12 @@
     {% for a in args.add_activity  %}
     <activity android:name="{{ a }}"></activity>
     {% endfor %}
+    <receiver android:name="com.eroad.uploader.DashcamuploaderBootReceiver" >
```

Comment:
Where does DashcamuploaderBootReceiver reside in the current PR?

Reply:
Yeah I know. It is in the gen3-cam-python. It is [here](https://github.com/eroad/gen3-cam-python/blob/b30d463d6ac9600e6f0a3affea8b1c04cd575e90/src/com/eroad/uploader/DashcamuploaderBootReceiver.java). I didn't want the py files to be referred here in this project. Also see this [discussion](https://github.com/eroad/gen3-cam-python/pull/9#discussion_r443926028)

Reply:
If you include receiver which is Java in the other repo it would make sense we do the same with the service and just alter the manifest on this repo so that changes are kept to a very minimal for maintenance over time.

Reply:
I had a discussion with Thay about this. On 2nd thought, i would move DashcamuploaderBootReceiver to this repo.

We can't move the DashCamPythonService because it is the used part of code generation and doesn't fit in the buildozer lifecycle if it stays outside.

Reply:
> I had a discussion with Thay about this. On 2nd thought, i would move DashcamuploaderBootReceiver to this repo.
> 
> We can't move the DashCamPythonService because it is the used part of code generation and doesn't fit in the buildozer lifecycle if it stays outside.

I tried to move the DashcamuploaderBootReceiver to this repo to keep them in one place, but the effort didn't pay off. So moved it back to gen3-cam-python. Created ticket https://jira.eroad.io/browse/DASH-434 to handle it later.